tests/vmi_configuration_test.go

```diff
@@ -477,8 +475,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 
 		Context("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with no memory requested", func() {
 			It("[test_id:3113]should failed to the VMI creation", func() {
-				vmi := tests.NewRandomVMI()
-				vmi.Spec.Domain.Resources = v1.ResourceRequirements{}
+				vmi := libvmi.New(withNoResourcesRequirements())
```

Comment:
I do not think you need `withNoResourcesRequirements()` here.

Reply:
You're right, it's not needed.
Removed.